channels/rdpdr/server/rdpdr_main.c

```diff
@@ -58,12 +101,16 @@ static int rdpdr_server_receive_announce_response(RdpdrServerContext* context, w
 	UINT32 ClientId;
 	UINT16 VersionMajor;
 	UINT16 VersionMinor;
+
```

Comment:
Not related to your changes but the available remaining size should be checked before reading